### PR TITLE
Added final solution for hw1

### PR DESCRIPTION
**Pull request by RuslanLyubimov**

### CODE REVIEW

**Comment by FakhriddinAbdurakhimovEPAM** [comment]
Can you add readme file with homework description, related links

**Comment by RuslanLyubimov** [comment]
Added Readme.md

File: hw1/task2.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const fs = require("fs");
+const csv = require("csvtojson");
+
+const CONSTANTS = require("./constants");
+
+const writeOutput = fs.createWriteStream("output.txt");
+const source = fs
```
Comment (FakhriddinAbdurakhimovEPAM) [inline hw1/task2.js]: you don't need source variable, just call fs

File: hw1/task3.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import csv from "csvtojson";
+import fs from "fs";
+import CONSTANTS from "./constants";
+
+const writeOutput = fs.createWriteStream("output2.txt");
+const source = fs
```
Comment (FakhriddinAbdurakhimovEPAM) [inline hw1/task3.js]: you don't need source variable, just call fs

**Review by FakhriddinAbdurakhimovEPAM — CHANGES_REQUESTED** [review]

File: hw1/task2.js (hunk 1 above)
Comment (RuslanLyubimov) [reply hw1/task2.js]: Fixed!

File: hw1/task3.js (hunk 2 above)
Comment (RuslanLyubimov) [reply hw1/task3.js]: Fixed!

**Review by FakhriddinAbdurakhimovEPAM — APPROVED** [review]

**Pull request merged**